samples/apps/github-qna-webapp-react/src/components/GitHubRepoSelection.tsx

```diff
@@ -122,9 +123,19 @@ const GitHubProjectSelection: FC<IData> = ({ uri, keyConfig, prevProject, prevBr
                     onClick={() => download()}
                 />
             </div>
-            <Label>
-                <strong>Embedding Filetypes</strong>
-            </Label>
+            <div style={{ display: 'flex', flexDirection: 'row' }}>
+                <Label>
+                    <strong>Embedding File Types</strong>
+                </Label>
+                <InfoLabel
+                    info={
+                        <div style={{ maxWidth: 250 }}>
+                            Only files of type below will be used for embedding and for answers.
```

Comment:
what about "Embedding and answers will be based on files of the type indicated below" ?

Reply:
looks good